libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -141,6 +141,46 @@ void AP_Camera_Backend::send_camera_feedback(mavlink_channel_t chan)
         camera_feedback.feedback_trigger_logged_count); // completed image captures
 }
 
+// send camera information message to GCS
+void AP_Camera_Backend::send_camera_information(mavlink_channel_t chan) const
+{
+    // prepare vendor, model and cam definition strings
+    const uint8_t vendor_name[32] {};
+    const uint8_t model_name[32] {};
+    const char cam_definition_uri[140] {};
+
+    // send CAMERA_INFORMATION message
+    mavlink_msg_camera_information_send(
+        chan,
+        AP_HAL::millis(),       // time_boot_ms
+        vendor_name,            // vendor_name uint8_t[32]
+        model_name,             // model_name uint8_t[32]
+        0,                      // firmware version uint32_t
```

Comment:
See mavlink PRs for better default values for all of these.

I think we could reasonably fill a lot of these values in from json files on SD card or similar - once that jasmine parser thing is merged.

Reply:
OK, I've changed the defaults to be NaN for focal_length and sensor_size_h/v